Lesson_12/Tusk_3.py

```diff
@@ -0,0 +1,10 @@
+from time import ctime
+
+
+class MyCustomException(Exception):
+    with open('custom_exception_info.txt', 'w') as file:
```

Comment:
This should be put to the init method. And I would suggest to change modifier to `w+` to make sure user won't get an error if file does not exist
```
class MyCustomException(Exception):
    def __init__(self, message):
        with open('custom_exception_info.txt', 'w+') as file:
            file.write(f'The "{Exception}" exception arised at {ctime()}\n')

Reply:
Done. Thx, now it works as it should